recipes/micro/build.sh

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/env bash
+
+set -eoux pipefail
+
+
+make build
+make install
```

Comment:
Is there a make check/test?

Reply:
It seems so!
https://github.com/zyedidia/micro/blob/f1aaa307436f0ee3fbf0acc5b490cf328662de82/Makefile#L66-L68